LISA/Executor.cpp

```diff
@@ -188,14 +188,14 @@ uint32_t Executor::GetStorageDetails(const std::string& type,
             // In Stage 1 there will be only one entry here
             for(const auto& i: appsPaths)
             {
-                appUsedKB += fs::getDirectorySpace(i);
+                appUsedKB += fs::getDirectorySpace(fs::getAppsDir() + i);
```

Comment:
Maybe reverse the order and use:
appUsedKB += fs::getDirectorySpace(details.appPath);
instead of determining it twice?